.evergreen/build-mac.sh

```diff
@@ -8,22 +8,25 @@ rm -rf validdist
 mkdir -p validdist
 mv dist/* validdist || true
 
-for VERSION in 3.7 3.8 3.9 3.10; do
-    PYTHON=/Library/Frameworks/Python.framework/Versions/$VERSION/bin/python3
-    rm -rf build
+VERSION=${VERSION:-3.10}
+PYTHON=/Library/Frameworks/Python.framework/Versions/$VERSION/bin/python3
+rm -rf build
 
-    createvirtualenv $PYTHON releasevenv
-    python -m pip install --upgrade wheel
-    python -m pip install setuptools==63.2.0
-    python setup.py bdist_wheel
-    deactivate || true
-    rm -rf releasevenv
+createvirtualenv $PYTHON releasevenv
+python -m pip install --upgrade wheel
+python -m pip install setuptools==63.2.0
+python setup.py bdist_wheel
+deactivate || true
+rm -rf releasevenv
 
-    # Test that each wheel is installable.
-    for release in dist/*; do
-        testinstall $PYTHON $release
-        mv $release validdist/
-    done
+# Test that each wheel is installable.
+for release in dist/*; do
+    if [[ "$release" = "*universal2*'" ]]; then
+        testinstall $PYTHON --ensure-universal2
```

Comment:
testinstall doesn't accept a `--ensure-universal2` param. I wonder if it would make more sense to set an ENV var in the release-mac-1100 tasks rather then try to derive if we should be creating a universal2 wheel after the fact. Then we could use that ENV var in testinstall.

Reply:
Done

Reply:
I verified that the wheels are checked in the build.